.github/workflows/release.yaml

```diff
@@ -87,13 +92,12 @@ jobs:
         id: meta
         uses: docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
         with:
-          images: |
-            ${{ env.dockerhub_image_name }}
-            ${{ env.ghcr_image_name }}
+          images: ${{ matrix.image }}
       - name: Build and push
         id: docker_build
         uses: docker/build-push-action@v2
         with:
+          file: ${{ matrix.dockerfile }}
           context: .
           platforms: linux/amd64,linux/arm64,linux/arm
           push: true
```

Comment:
Same for the ghcr signing

```
            COSIGN_REPOSITORY: ${{ env.ghcr_image_name }}/signs
```

Reply:
good catch. Totally agree

Reply:
How do you suggest to remediate that? Should I keep the env variable? We would not want to loop through the matrix again just for the ghrc/controller right?

Reply:
keep the env variable, yes. I tried to use the env variable in the matrix but I reproduced errors. So keep that matrix as I shared with you and keep the env variables , please. Thanks again

Reply:
ok done, thanks a lot for the review!